README.md

```diff
@@ -2,18 +2,15 @@
 
 > Desktop version
 
-> This project is a responsive version website.
+> This project is a responsive version website with javascript features
 
 Additional description about the project and its features.
```

Comment:
[ OPTIONAL ] Please add the details about project features here.  It's very useful for project users to know what is the project about.